### PR TITLE
Linter

### PR DESCRIPTION
**Pull request by lcd1232**
Added support for golangci-lint

### CODE REVIEW

**Comment by CryoByte33** [comment]
Would you mind rebasing this on the new `develop` branch? Trying to follow proper git flow and not break things for everyone 🙂

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+on: [push]
```
Comment (CryoByte33) [inline .github/workflows/lint.yml:1]: Should lint.yml be part of release.yml and have build depend on linting? I think that would make more sense and prevent bad code from making it in.

Comment (lcd1232) [reply .github/workflows/lint.yml:1]: I'm not sure that it does matter to do it on release. You can just forbid users to merge PR if the linter step is failed

Comment (CryoByte33) [reply .github/workflows/lint.yml:1]: My thought was that I'd not want a dev release to build if there are linter failures, which can only be done from the same pipeline if I'm not mistaken 🤔

**Pull request closed without merging**